deeppavlov/models/preprocessors/dirty_comments_preprocessor.py

```diff
@@ -57,7 +58,22 @@ def __call__(self, batch: List[str], **kwargs) -> List[str]:
         f = [re.sub("ies( |$)", "y ", x) for x in f]
         f = [re.sub("s( |$)", " ", x) for x in f]
         f = [re.sub("ing( |$)", " ", x) for x in f]
-        f = [x.replace("tard ", " ") for x in f]
+
+        f = [x.replace(" u ", " you ") for x in f]
+        f = [x.replace(" em ", " them ") for x in f]
+        f = [x.replace(" da ", " the ") for x in f]
+        f = [x.replace(" yo ", " you ") for x in f]
+        f = [x.replace(" ur ", " your ") for x in f]
+        f = [x.replace(" u r ", " you are ") for x in f]
+        f = [x.replace("yo\'re", "you are") for x in f]
+        f = [x.replace("yu\'re", "you are") for x in f]
+        f = [x.replace("u\'re", "you are") for x in f]
```

Comment:
`"you're".replace("u\'re", "you are")` → `'yoyou are'`

Reply:
this will be preprocessed above `f = [x.replace("you're ", "you are ") for x in f]`

Reply:
Then `f = [x.replace("'re", " are") for x in f]` above will make this rule never work

Reply:
yes, will change it